.circleci/config.yml

```diff
@@ -248,7 +248,7 @@ jobs:
           name: "Build new package upgrading from << parameters.package >>@<< parameters.preset >> and keep RPC running"
           background: true
           command: |
-            yarn cannon build << parameters.toml >> \
+            NODE_OPTIONS=--max_old_space_size=8192 yarn cannon build << parameters.toml >> \
```

Comment:
@noisekit please, confirm this is correct or is a better way to do it.

Reply:
I think it is fine. But surprised this is necessary. Why do we need it?